### PR TITLE
Update mcf_format.md

### PR DESCRIPTION
**Pull request by spiekos**
Added additional documentation about formatting of tMCF

### CODE REVIEW

**Review by beets — COMMENTED** [review]
please also include the image file referenced here:
img/example_schema_mapping_diagram_2.png

File: docs/mcf_format.md [hunk 1]
```diff
@@ -316,7 +318,34 @@ typeOf: SurveyItem
 dcid: C:SomeDataset->SurveyItem_Dcid
 ```
 
-Where E is short for "Entity" and "C" is short for "Column".
+Where "E" is short for "Entity" and "C" is short for "Column".
+
+"E" should be used to refrence another entity that is defined in the template
+MCF only. Instead "C" should be used to refer to property values that are 
+entities already populated in the graph and not being defined in the template
```
Comment (pradh) [inline docs/mcf_format.md:325]: nit: minor reword suggestion... feel free to ignore :)

Instead "C" should be used to refer to property values represented in columns of the `.csv` file. The values may include:
* References to entities that are already in the graph, and not defined in the Template MCF
* Constants (strings, numbers)
* Enumerations defined in the schema
* Quantities, like [10 20 Years]

File: docs/mcf_format.md [hunk 2]
```diff
@@ -316,7 +318,34 @@ typeOf: SurveyItem
...
 ```
 
-Where E is short for "Entity" and "C" is short for "Column".
+Where "E" is short for "Entity" and "C" is short for "Column".
+
+"E" should be used to refrence another entity that is defined in the template
+MCF only. Instead "C" should be used to refer to property values that are 
+entities already populated in the graph and not being defined in the template 
+MCF. "C" should also be used to reference property values that are Quantities
+or Enummerations already defined in the schema.
```
Comment (pradh) [inline docs/mcf_format.md:327]: nit: enumerations

File: docs/mcf_format.md [hunk 3]
```diff
@@ -288,10 +288,12 @@ other nodes can be `typeOf`.
 
 ### Template MCF
 
-Template MCF is an intermediary MCF that we use for mapping datasets to instance
-MCF. Template MCF does not get ingested into the Data Commons graph.
+Template MCF, or tMCF, is an intermediary MCF that we use for mapping datasets to instance
+MCF. Template MCF does not get ingested into the Data Commons graph. They are
+referencing the data in a paired .csv and together will be used to generate a
```
Comment (pradh) [inline docs/mcf_format.md:293]: nit: "They are referencing the data in a" -> "Template MCF references data in a"

**Review by pradh — COMMENTED** [review]
Thanks for the improvements!

**Review by spiekos — COMMENTED** [review]
Updated the MCFs with the requested changes, added the schema drawing, and included more information on property values of type Quantity. Please, check the schema diagram to double check that it makes sense.

File: docs/mcf_format.md [hunk 4]
```diff
@@ -316,7 +318,43 @@ typeOf: SurveyItem
...
+Node: E:SomeDataset->E1
+typeOf: City
+dcid: C:SomeDataset->City_Dcid
+name: C:SomeDataset->City_Name
+containedInPlace C:SomeDataset->State_Dcid
+waterArea: C:SomeDataset->Water_Area
+citySize: C:SomeDataset->City_Size
+```
+
+A note about property values that are quantities like "waterArea" is in this
```
Comment (beets) [inline docs/mcf_format.md:354]:
```suggestion
A note about property values that are quantities (like "waterArea" is in this
```

File: docs/mcf_format.md [hunk 5]
```diff
@@ -316,7 +318,43 @@ typeOf: SurveyItem
...
+example, the property value will be formated as [value unitOfMeasure]. This
```
Comment (beets) [inline docs/mcf_format.md:355]:
```suggestion
example): the property value will be formatted as [value unitOfMeasure]. This
```

File: docs/mcf_format.md [hunk 6]
```diff
@@ -316,7 +318,43 @@ typeOf: SurveyItem
...
+The template MCF looks like:
+
+```
+Node: E:SomeDataset->E1
+typeOf: City
+dcid: C:SomeDataset->City_Dcid
+name: C:SomeDataset->City_Name
+containedInPlace C:SomeDataset->State_Dcid
+waterArea: C:SomeDataset->Water_Area
+citySize: C:SomeDataset->City_Size
```
Comment (beets) [inline docs/mcf_format.md:351]: I'd suggest moving this up with the other city related values.

**Review by beets — APPROVED** [review]
thanks for the edits! some minor suggestions below

**Pull request merged**